draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1568,11 +1568,11 @@ algorithm documented in Sections {{MP_KEY}} and {{handshaking}}. The
 security of the MP-DCCP connection depends on the use of keys that are
 shared once at the start of the first subflow and are never sent again
 over the network. To ease demultiplexing while not revealing
-cryptographic material, future subflows use a truncated cryptographic
-hash of this key as the connection identification "token". The keys are
+cryptographic material, future subflows use the initially exchanged
+CI information. The keys exchanged once at the beginning are
 concatenated and used as keys for creating Hash-based Message
 Authentication Codes (HMACs) used on subflow setup, in order to verify
-that the parties in the handshake are the same as in the original
+that the parties in the handshake of future subflows are the same as in the original
```

Comment:
"future subflows" sound a bit strange to me here. "of the new subflow" or "of additional subflows"?

Reply:
changed to "subsequent subflows" in https://github.com/tsvwg/ietf-multipath-dccp/pull/269/commits/3e95e62dd8402ba0021c267fd4020f4a7f04dfd1